mmv1/third_party/terraform/services/compute/resource_compute_instance_from_template_test.go.tmpl

```diff
@@ -1482,6 +1509,56 @@ resource "google_compute_instance_from_template" "inst" {
 `, templateDisk, template, instance)
 }
 
+func testAccComputeInstanceFromTemplate_terminationTime(templateDisk, template, termination_time, instance string) string {
+	return fmt.Sprintf(`
+data "google_compute_image" "my_image" {
+  family  = "debian-11"
+  project = "debian-cloud"
+}
+
+resource "google_compute_disk" "foobar" {
+  name  = "%s"
+  image = data.google_compute_image.my_image.self_link
+  size  = 10
+  type  = "pd-ssd"
+  zone  = "us-central1-a"
+}
+
+resource "google_compute_instance_template" "foobar" {
+  name         = "%s"
+  machine_type = "e2-medium"
+
+  disk {
+    source      = google_compute_disk.foobar.name
+    auto_delete = false
+    boot        = true
+  }
+
+  network_interface {
+    network = "default"
+  }
+
+  metadata = {
+    foo = "bar"
+  }
+
+  scheduling {
+    instance_termination_action = "STOP"
+    termination_time = "%s"
+  }
+
+  can_ip_forward = true
+}
+
+resource "google_compute_instance_from_template" "inst" {
```

Comment:
This resource needs to override the termination_time.

(The `testAccComputeInstanceFromTemplate_overrideScheduling` step should have added a similar override for its scheduling fields but apparently the reviewer missed asking them to add it. No need to fix that since this is already a large change.)

Reply:
This resource (google_compute_instance_from_template.inst) still needs to override `scheduling.termination_time` - this is why the automation is detecting `google_compute_instance_from_template.scheduling.termination_time` as untested: https://github.com/GoogleCloudPlatform/magic-modules/pull/12791#issuecomment-2675165487

Reply:
I added overwrite.